umap/static/umap/js/umap.forms.js

```diff
@@ -939,6 +939,10 @@ L.U.FormBuilder = L.FormBuilder.extend({
 
   defaultOptions: {
     name: { label: L._('name') },
+    owner: {
+      label: 'owner',
```

Comment:
```suggestion
      label: L._('owner'),
```

Useful to be able to translate user-facing strings.